ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -219,6 +219,24 @@ objects:
         trap 'touch /tmp/shared/exit' EXIT
         trap 'kill $(jobs -p); exit 0' TERM
 
+        function fips_check() {
+          oc --insecure-skip-tls-verify --request-timeout=60s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodes
+          while read p; do
```

Comment:
It would be simpler to use a variable and a foreach:
```bash
          nodes=($(oc --insecure-skip-tls-verify --request-timeout=60s get nodes -o name))
          for node in "${nodes[@]}"; do
            # ...
```